pymemcache/client/base.py

```diff
@@ -82,10 +82,10 @@ def _parse_hex(value):
 
 def _check_key(key, key_prefix=b''):
     """Checks key and add key_prefix."""
-    if isinstance(key, six.text_type):
+    if isinstance(key, six.text_type) or isinstance(key, six.string_types):
```

Comment:
We might want to introduce a top-level variable that defines the set of allowed key types so that this can become a single `isinstance` check:

```python
key_types = six.string_types + (six.text_types,) # or a similar concatenation

if isinstance(key, key_types):
```